examples/nrf52840_dogm132w5_graphics.rs

```diff
@@ -48,25 +52,29 @@ fn main() -> ! {
     let disp_si = port0.p0_19.into_push_pull_output(Level::Low).degrade();
 
     // Create DOGM132W-5 spi bus
-    let disp_spi = SPIInterface::new(
-        hal::Spim::new(
-            peripherals.SPIM0,
-            hal::spim::Pins {
-                sck: Some(disp_scl),
-                mosi: Some(disp_si),
-                miso: None,
-            },
-            hal::spim::Frequency::M8,
-            hal::spim::MODE_3,
-            0,
-        ),
-        disp_a0,
-        disp_cs,
+    let spi_bus = hal::Spim::new(
+        peripherals.SPIM0,
+        hal::spim::Pins {
+            sck: Some(disp_scl),
+            mosi: Some(disp_si),
+            miso: None,
+        },
+        hal::spim::Frequency::M8,
+        hal::spim::MODE_3,
+        0,
     );
 
+    // Create an ExclusiveDevice, the bus will be owned only by the device
+    let disp_device = ExclusiveDevice::new_no_delay(spi_bus, disp_cs).unwrap();
+
+    // if you need to share the spi bus, create a shared bus and a shared device (AtomicDevice)
+    // let atomic_spi_bus = AtomicCell::new(spi_bus);
+    // let disp_device = AtomicDevice::new(&atomic_spi_bus, disp_cs, hal::timer::Timer::new(peripherals.TIMER1)).unwrap();
```

Comment:
Use the no_delay version here as well

Reply:
Actually, I would just remove the entire AtomicDevice code. Documenting bus sharing is responsibility of `embedded_hal_bus`, not of this crate.